src/Hypergrid.js

```diff
@@ -1255,12 +1255,24 @@ var Hypergrid = Base.extend('Hypergrid', {
         return window.open.apply(window, arguments);
     },
 
-    getColumns: function() {
-        return this.behavior.getColumns();
+    /**
+     * @param {number} [begin]
+     * @param {nubmer} [end]
+     * * @returns {Column[]} A copy of the all columns array by passing the params to `Array.prototype.slice`.
+     */
+    getColumns: function(begin, end) {
```

Comment:
Both of these functions are relatively new (added last week). They now return copies. If you want the originals, call the `behavior` versions.